test/integration_test.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib
+
+import pytest
+from hydra import compose, initialize
+from omegaconf import DictConfig, OmegaConf
+
+system_paths = [
+    "ppo.ff_ippo",
+    # "ppo.ff_mappo",  # don't use this because ff_mappo is used in `test_discrete_env`
+    "ppo.rec_ippo",
+    "ppo.rec_mappo",
+    "sac.ff_isac",
+    "sac.ff_masac",
+    "q_learning.rec_iql",
+]
+
+discrete_envs = [
+    "gigastep",
+    "lbf",
+    "matrax",
+    # "rware",  # don't use this because `test_mava_system` will run all systems on rware
+    "smax",
+]
+cnn_envs = ["cleaner", "connector"]
+continuous_envs = ["mabrax"]
+
+
+def _run_system(system_name: str, cfg: DictConfig) -> float:
+    """Runs a system."""
+    OmegaConf.set_struct(cfg, False)
+    cfg.logger.use_neptune = False  # we never want to log these tests to neptune
```

Comment:
```suggestion
    # we never want to log these tests anywhere
    cfg.logger.use_neptune = False
    cfg.logger.use_tb = False
    cfg.logger.use_json = False
```

We can do the same for tensorboard and json logging as well here please.